script-test/playbackstrategies/modifiers/live/seekabletest.js

```diff
@@ -326,6 +339,22 @@ require(
 
           expect(player.resume).not.toHaveBeenCalledTimes(1);
         });
+
+        it('Should auto resume when paused after a seek', function () {
+          player.getSeekableRange.and.returnValue({start: 0});
+          player.getCurrentTime.and.returnValue(100);
+
+          startPlaybackAndPause(100, false);
+
+          player.getCurrentTime.and.returnValue(50);
+          player.getState.and.returnValue(MediaPlayerBase.STATE.PAUSED);
```

Comment:
Shouldn't this be a call to `pause` after calling `playFrom`  rather than forcing the state?

Reply:
HTML5 Player is being spied on, pause isn't triggering state changes.

Reply:
As discussed, this sort of change would be difficult given the state the tests are in, so we'll be pragmatic rather than perfectionist.